features/machine/machine_healthcheck.feature

```diff
@@ -45,3 +45,34 @@ Feature: MachineHealthCheck Test Scenarios
     And I ensure "mhc-<%= machine_set.name %>" machinehealthcheck is deleted after scenario
 
     Then the machine should be remediated
+
+  # @author jhou@redhat.com
+  # @case_id OCP-25734
+  @admin
+  @destructive
+  Scenario: Create multiple MHCs to monitor same machineset
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I clone a machineset named "machineset-25734"
+
+    # Create MHC
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/mhc/mhc1.yaml" replacing paths:
+      | n                                                                                  | openshift-machine-api         |
+      | ["metadata"]["name"]                                                               | mhc-<%= machine_set.name %>-0 |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]    | <%= machine_set.cluster %>    |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"] | <%= machine_set.name %>       |
+    Then the step should succeed
+    And I ensure "mhc-<%= machine_set.name %>-0" machinehealthcheck is deleted after scenario
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/mhc/mhc1.yaml" replacing paths:
+      | n                                                                                  | openshift-machine-api         |
+      | ["metadata"]["name"]                                                               | mhc-<%= machine_set.name %>-1 |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]    | <%= machine_set.cluster %>    |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"] | <%= machine_set.name %>       |
+    Then the step should succeed
+    And I ensure "mhc-<%= machine_set.name %>-1" machinehealthcheck is deleted after scenario
```

Comment:
For duplicate steps, we can use `I run the steps xxx times`

Reply:
Thanks, updated